src/Parser.hs

```diff
@@ -357,10 +352,8 @@ parseFunc =
     (params,typ) <- parseFuncSignature
     DT.traceM $ "Func signature parts are " ++ show params ++ " and " ++ show typ
     reservedOp "="
-    reservedOp "{"
-    expr <- parseExpr
-    reservedOp "}"
-    let f = Func name params typ (Body expr "")
+    expr <- braces $ many1 parseExpr
```

Comment:
Dang, I missed a call to `parseBlock` here.

Reply:
We can probably add this in here then, would be good to keep the usage consistent.